src/main/java/hu/blackbelt/structured/map/proxy/MapProxy.java

```diff
@@ -54,6 +54,8 @@ public final class MapProxy implements InvocationHandler {
 
     public static final String METHOD_SET = "set";
     public static final String METHOD_GET = "get";
+    public static final String METHOD_ADD = "addTo";
```

Comment:
add, and remove should be fine

Reply:
When generating SDK code, addTo and removeFrom make function names more readable.